ansible-scylla-node/templates/cqlshrc.j2

```diff
@@ -1,6 +1,11 @@
 [authentication] ;; Change these after the user/password has been updated from the default
+{% if cql_username is defined and cql_password is defined %}
+username = {{ cql_username }}
+password = {{ cql_password }}
+{% else %}
```

Comment:
How this patch is related to this PR?

Reply:
At first I though it would be a good idea to use these new credentials to update cqlshrc as well, but now I see that it doesn't make sense.
Patch removed.